py/torch_tensorrt/dynamo/conversion/impl/reduce.py

```diff
@@ -33,3 +33,29 @@ def amax(
     )
     set_layer_name(layer, target, name, source_ir)
     return layer.get_output(0)
+
+
+def sum_dim_IntList(
```

Comment:
this seems over specialized, what are the other variants of sum? Can we combine the implementations?

Reply:
@narendasan Thanks for your suggestion! I updated to support both `aten.sum.default` and `aten.sum.dim_IntList`.

Reply:
@narendasan lmk if more changes are needed! thanks!